cli/popper/runner_slurm.py

```diff
@@ -151,3 +152,72 @@ def _create_cmd(self, step, img, cid):
         cmd.append(f'{image} {command}')
 
         return ' '.join(cmd)
+
+
+class SingularityRunner(SlurmRunner, HostSingularityRunner):
```

Comment:
very nice, I really like this multi-inheritance approach, allows us to keep the code succint and readable